src/shared/sources/pl/index.js

```diff
@@ -0,0 +1,84 @@
+const parse = require('../_lib/parse.js')
+const maintainers = require('../_lib/maintainers.js')
+
+const isoMap = require('./mapping.json')
+
+const country = "iso1:PL"
+
+module.exports = {
+    country,
+    maintainers: [ maintainers.qgolsteyn, maintainers.ciscorucinski ],
+    friendly: {
+        name: 'Ministry of Health of the Republic of Poland',
+        url: 'http://www.mz.gov.pl/'
+    },
+    scrapers: [
+        // TODO (scraper) PL: 2020-03-18 scraper
+        // {
+        //     startDate: '2020-03-18',
+        //     crawl: [
+        //         {
+        //             type: 'csv',
+        //             url: 'https://raw.githubusercontent.com/covid19-eu-zh/covid19-eu-data/master/dataset/covid-19-pl.csv'
+        //         }
+        //     ],
+        //     scrape (data, date) {
+        //         const casesByRegion = {}
+        //         const deathsByRegion = {}
+        //
+        //         for (const item of data) {
+        //             const {datetime, nuts_2} = item;
+        //             if (datetime.dateIsBeforeOrEqualTo(datetime, date) && nuts_2) {
+        //                 casesByRegion[nuts_2] = parse.number(item.cases)
+        //                 deathsByRegion[nuts_2] = parse.number(item.deaths)
+        //             }
+        //         }
+        //
+        //         const states = []
+        //         for (const region of Object.keys(casesByRegion)) {
+        //             states.push({
+        //                 state: mapping[region],
+        //                 cases: casesByRegion[region]
+        //             })
+        //         }
+        //
+        //         if (states.length > 0) states.push(transform.sumData(states))
+        //         return states
+        //     }
+        // },
+        {
+            startDate: '2020-04-13',
+            crawl: [
+                {
+                    type: 'page',
+                    url: 'https://www.gov.pl/web/koronawirus/wykaz-zarazen-koronawirusem-sars-cov-2',
+                }
+            ],
+            // eslint-disable-next-line no-unused-vars
+            scrape ($, date, { getIso2FromName }) {
+                console.log("New Scraper")
+                const $pre = $('#registerData')
+                const casesData = JSON.parse(JSON.parse($pre.text()).parsedData)
+
+                const casesByRegion = {}
+                const deathsByRegion = {}
+
+                for (const item of casesData) {
```

Comment:
May be preferable to iterate through the items and make our `data` in one loop (more similar to on line 72) than looping through here, and then again in 72?

Reply:
Can do this later.  Cheers!